crates/dids/methods/web/src/lib.rs

```diff
@@ -125,6 +125,7 @@ impl DIDMethodResolver for DIDWeb {
             .get(header::CONTENT_TYPE)
             .map(|value| match value.as_bytes() {
                 b"application/json" => Ok(MediaType::Json),
+                b"application/json; charset=utf-8" => Ok(MediaType::Json),
```

Comment:
Should probably split on `;` to handle those cases if we're not going to just use a library to parse the content type header